languages/javalang/builder-renderer/commercetools-java-client/src/main/java/client/TokenIntrospection.java

```diff
@@ -0,0 +1,38 @@
+package client;
+
+public class TokenIntrospection {
```

Comment:
Looks like we don't use this yet. Do you think we will use it later? If not I would suggest to remove this class.

Reply:
we use it : class TempClient line 118.